mender-deb-package

```diff
@@ -50,6 +50,24 @@ checkout_repo() {
   git checkout $VERSION
 }
 
+install_go() {
+ local GOLANG_VERSION=1.14.7
+ local golang_version_set
+
+ golang_version_set=$(sed -n '/[^ ]*GOLANG_VERSION:[ "0-9]/{s/.*GOLANG_VERSION:[ ]*\(["0-9\.]*\).*/\1/p;q}' .gitlab-ci.yml)
+ golang_version_set=${golang_version_set//\"/}
+ GOLANG_VERSION=${golang_version_set:-${GOLANG_VERSION}}
+
+ wget -q https://dl.google.com/go/go$GOLANG_VERSION.linux-amd64.tar.gz \
+    && tar -C /usr/local -xzf go$GOLANG_VERSION.linux-amd64.tar.gz
+ export GOPATH="/root/go"
+ export PATH="$PATH:/usr/local/go/bin"
+ # Support building mender-client 2.3.x, since it does not have go modules support
+ # For newer clients with a go.mod file, this is a no-op however.
+ export GO111MODULE=auto
```

Comment:
This we can remove, really. 2.3.x has been EOL for a long while now

Reply:
removed.